src/subscribables/observable.js

```diff
@@ -25,8 +25,13 @@ ko.observable = function (initialValue) {
 
     if (DEBUG) observable._latestValue = _latestValue;
     observable.peek = function() { return _latestValue };
-    observable.valueHasMutated = function () { observable["notifySubscribers"](_latestValue); }
-    observable.valueWillMutate = function () { observable["notifySubscribers"](_latestValue, "beforeChange"); }
+    observable.valueHasMutated = function () {
+        observable.updateVersion();
```

Comment:
Do you think the above line should perhaps be moved inside `notifySubscribers`? The version number would then mean "notification number" rather than "mutation number".

The reason I suggest this is that I think there's a lot of code out in the wild that does stuff like this:

    myObservable().subProperty = "New value";
    myObservable.notifySubscribers();

I know that's not the recommended pattern, but people may do it (and in practice, there are sure to be plenty of cases where it is done).

If the version continues to refer to mutations rather than notifications, pure computeds won't update correctly in this case, will they?